### PR TITLE
Feature/cu 86dvfm501/initial structure ecommerce

### PR DESCRIPTION
**Pull request by tomisueldo**
# ⚡ Initial Structure Ecommerce - [CU-86dvfm501](https://app.clickup.com/t/86dvfm501) ⚡

## 💻 What type of change is this?

- [x] 💎 Feature

## ⭐ Description

I have completed the first challenge in HTML and CSS. The key features include:

- A product listing.
- The implementation of CSS for styling.

## 📷 Screenshots

image

### To review
- [ ] I have tested this change locally in multiple screen sizes
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] If my task include an endpoint, I add the endpoint to Hopscotch/Postman Project
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by mateorodriguezcastillo — APPROVED** [review]

File: .idea/vcs.xml
Comment (SantiagoVitureiraLightIt) [inline .idea/vcs.xml:1]: This is an autogenereted file, please add this to .gitignore

File: index.html [hunk 1]
```diff
@@ -17,56 +17,79 @@
...
       <h2>Wooyoungmi</h2>
       <p>Dark-grey slub wool, pintucked notch lapels.</p>
-      <span class="price">€ 555</span>
-      <button class="favorite"><3</button>
+      <div class="bottom-description">
+        <span class="price">€ 555</span>
+        <i class="fa-regular fa-heart fa-beat"></i>
+      </div>
     </div>
+
```
Comment (SantiagoVitureiraLightIt) [inline index.html:69]: This is your only line between tags, please be consistent

File: img/thom_sweeney.png
Comment (SantiagoVitureiraLightIt) [inline img/thom_sweeney.png:1]: Please change the name to kebab-case

File: img/tom_ford.png
Comment (SantiagoVitureiraLightIt) [inline img/tom_ford.png:1]: Please change the name to kebab-case

**Review by SantiagoVitureiraLightIt — COMMENTED** [review]
Small comments, very nice work!

**Pull request merged**